boards.txt

```diff
@@ -9895,6 +9895,91 @@ m5stack-fire.menu.DebugLevel.verbose.build.code_debug=5
+m5stack-station.build.core=esp32
+m5stack-station.build.variant=m5stack_station
+m5stack-station.build.board=M5Stack_Station
+
+m5stack-station.build.f_cpu=240000000L
+m5stack-station.build.flash_size=16MB
+m5stack-station.build.flash_freq=80m
+m5stack-station.build.flash_mode=dio
+m5stack-station.build.boot=dio
+m5stack-station.build.partitions=default_16MB
+m5stack-station.build.defines=
+
+m5stack-station.menu.PartitionScheme.default=Default
+m5stack-station.menu.PartitionScheme.default.build.partitions=default_16MB
+m5stack-station.menu.PartitionScheme.no_ota=No OTA (Large APP)
+m5stack-station.menu.PartitionScheme.no_ota.build.partitions=no_ota
+m5stack-station.menu.PartitionScheme.no_ota.upload.maximum_size=6553600
+m5stack-station.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (Large APPS with OTA)
+m5stack-station.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+m5stack-station.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+
+m5stack-station.menu.CPUFreq.240=240MHz (WiFi/BT)
+m5stack-station.menu.CPUFreq.240.build.f_cpu=240000000L
+m5stack-station.menu.CPUFreq.160=160MHz (WiFi/BT)
+m5stack-station.menu.CPUFreq.160.build.f_cpu=160000000L
+m5stack-station.menu.CPUFreq.80=80MHz (WiFi/BT)
+m5stack-station.menu.CPUFreq.80.build.f_cpu=80000000L
+m5stack-station.menu.CPUFreq.40=40MHz (40MHz XTAL)
+m5stack-station.menu.CPUFreq.40.build.f_cpu=40000000L
+m5stack-station.menu.CPUFreq.26=26MHz (26MHz XTAL)
+m5stack-station.menu.CPUFreq.26.build.f_cpu=26000000L
+m5stack-station.menu.CPUFreq.20=20MHz (40MHz XTAL)
+m5stack-station.menu.CPUFreq.20.build.f_cpu=20000000L
+m5stack-station.menu.CPUFreq.13=13MHz (26MHz XTAL)
+m5stack-station.menu.CPUFreq.13.build.f_cpu=13000000L
+m5stack-station.menu.CPUFreq.10=10MHz (40MHz XTAL)
+m5stack-station.menu.CPUFreq.10.build.f_cpu=10000000L
+
+m5stack-station.menu.UploadSpeed.1500000=1500000
+m5stack-station.menu.UploadSpeed.1500000.upload.speed=1500000
+m5stack-station.menu.UploadSpeed.750000=750000
+m5stack-station.menu.UploadSpeed.750000.upload.speed=750000
+m5stack-station.menu.UploadSpeed.500000=500000
+m5stack-station.menu.UploadSpeed.500000.upload.speed=500000
+m5stack-station.menu.UploadSpeed.250000=250000
+m5stack-station.menu.UploadSpeed.250000.upload.speed=250000
+m5stack-station.menu.UploadSpeed.115200=115200
+m5stack-station.menu.UploadSpeed.115200.upload.speed=115200
+
+m5stack-station.menu.DebugLevel.none=None
+m5stack-station.menu.DebugLevel.none.build.code_debug=0
+m5stack-station.menu.DebugLevel.error=Error
+m5stack-station.menu.DebugLevel.error.build.code_debug=1
+m5stack-station.menu.DebugLevel.warn=Warn
+m5stack-station.menu.DebugLevel.warn.build.code_debug=2
+m5stack-station.menu.DebugLevel.info=Info
+m5stack-station.menu.DebugLevel.info.build.code_debug=3
+m5stack-station.menu.DebugLevel.debug=Debug
+m5stack-station.menu.DebugLevel.debug.build.code_debug=4
+m5stack-station.menu.DebugLevel.verbose=Verbose
+m5stack-station.menu.DebugLevel.verbose.build.code_debug=5
+
```

Comment:
Can you please add the Erase_Flash menu option, its needed for all boards from PR #7043 and PR #7129 is adding that to all existing boards.

Reply:
Thank you for your feedback, I will add in time.

Reply:
Already taken care of

Reply:
Thanks